docs/cubeviz/plugins.rst

```diff
@@ -114,58 +117,20 @@ the above figure.
 Model Fitting
 =============
 
-The Model Fitting plugin is described in more detail by the
-:ref:`Specviz Model Fitting <specviz-model-fitting>` documentation.
-For Cubeviz, there is an additional option to fit the model over each individual spaxel by
-enabling the :guilabel:`Cube Fit` toggle before pressing :guilabel:`Fit Model`.  
-The fit parameter planes are saved in a data structure
-that can be accessed in a jupyter notebook. The best model fit (evaluated over the cube) is
-also saved to a dataset with the label specified in the :guilabel:`Output Data Label` field.
-
 .. seealso::
 
     :ref:`Model Fitting <specviz-model-fitting>`
         Specviz documentation on fitting spectral models.
 
-As mentioned in the :ref:`specviz-model-fitting` section for Specviz, models can be accessed
-in a Jupyter notebook by using the following code::
-
-    cubeviz.get_models(model_label="Model", x=10)
-
-Where the ``model_label`` keyword identifies which model should be returned and the ``x`` and
-``y`` keywords identify specifically which spaxel fits are to be returned, for models
-applied to every spaxel using the :guilabel:`Apply to Cube` button. Leaving ``x`` or
-``y`` as `None` will mean that the models fit to every spaxel across that axis will be
-returned. Model parameters can also be accessed using the following code::
-
-    cubeviz.get_model_parameters(model_label="Model", y=52)
-
-With the parameters following the same pattern as the
-:meth:`~jdaviz.core.helpers.ConfigHelper.get_models` method.
-
-Contours
```

Comment:
Is the removal of Contours here intentional?

Reply:
Yes, we don't have a contours plugin. Nor do I think it needs much of an explanation in the current format.